src/test/java/com/kobylynskyi/graphql/codegen/GraphQLCodegenAnnotationsTest.java

```diff
@@ -55,7 +55,7 @@ void generate_CustomAnnotationMappings() throws Exception {
         assertFileContainsElements(files, "Event.java",
                 "@com.fasterxml.jackson.databind.annotation.JsonDeserialize(" +
                         "using = com.example.json.DateTimeScalarDeserializer.class)"
-                        + System.lineSeparator() + "    private org.joda.time.DateTime createdDateTime;");
+                        + "\n    private org.joda.time.DateTime createdDateTime;");
```

Comment:
Why should we modify it?

Reply:
This is current behavior, on windows your codegen always generates `\n`, not `\r\n`. I just made tests consistent with this behavior

Reply:
Agree with @jxnu-liguobin. I don't think we need to modify line separators.

Reply:
@kobylynskyi then you should update library code, because tests are failing on Windows now. Everyone who uses your library gets `\n` not `System.lineSeparator()`.
Please, hear me, I do not suggest to "modify line separators", I suggest to make tests reflecting current behavior

Reply:
@romash1408, sorry for a late reply. I've fixed all existing tests, so please pull the latest develop branch.

Reply:
@romash1408 I've resolved merge conflicts and now test on Windows should pass.

Reply:
@kobylynskyi thank you! Yes, all tests pass on windows